### ISSUE TITLE
WARNING: Contrast ratio of #fff on #13ce66 is pretty bad, just 2

### ISSUE THREAD

**Issue by pranavrajs** [chatwoot/chatwoot#116]
```
16:42:10 frontend.1 | WARNING: Contrast ratio of #fff on #13ce66 is pretty bad, just 2
16:42:10 frontend.1 |          on line 75 of node_modules/foundation-sites/scss/util/_color.scss, in function `color-pick-contrast`
16:42:10 frontend.1 |          from line 121 of node_modules/foundation-sites/scss/components/_button.scss, in mixin `button-style`
16:42:10 frontend.1 |          from line 203 of node_modules/foundation-sites/scss/components/_button-group.scss, in mixin `foundation-button-group`
16:42:10 frontend.1 |          from line 86 of node_modules/foundation-sites/scss/foundation.scss, in mixin `foundation-everything`
16:42:10 frontend.1 |          from line 3 of node_modules/foundation-sites/assets/foundation-flex.scss
16:42:10 frontend.1 |          from line 8 of app/javascript/src/assets/scss/app.scss
16:42:10 frontend.1 |          from line 28 of stdin
```

**Comment by pranavrajs** [comment]
```
16:42:10 frontend.1 | WARNING: Contrast ratio of #fff on #13ce66 is pretty bad, just 2
16:42:10 frontend.1 |          on line 75 of node_modules/foundation-sites/scss/util/_color.scss, in function `color-pick-contrast`
16:42:10 frontend.1 |          from line 121 of node_modules/foundation-sites/scss/components/_button.scss, in mixin `button-style`
16:42:10 frontend.1 |          from line 236 of node_modules/foundation-sites/scss/components/_button.scss, in mixin `foundation-button`
16:42:10 frontend.1 |          from line 81 of node_modules/foundation-sites/scss/foundation.scss, in mixin `foundation-everything`
16:42:10 frontend.1 |          from line 3 of node_modules/foundation-sites/assets/foundation-flex.scss
16:42:10 frontend.1 |          from line 8 of app/javascript/src/assets/scss/app.scss
16:42:10 frontend.1 |          from line 28 of stdin
```

**Comment by AndreiUngur** [comment]
Should be solved by https://github.com/chatwoot/chatwoot/pull/144

**Comment by sojan-official** [comment]
resolved in #144
